qml.qrc

```diff
@@ -241,5 +241,6 @@
         <file>images/edit.svg</file>
         <file>images/arrow-right-in-circle-outline-medium-white.svg</file>
         <file>images/tails-grey.png</file>
+        <file>images/appicons/64x64.png</file>
```

Comment:
Could you please move this before the line 239 ` <file>images/check-white.svg</file>`